packages/perseus/src/__tests__/renderer.test.tsx

```diff
@@ -15,6 +15,7 @@ import {
     definitionItem,
     mockedRandomItem,
     mockedShuffledRadioProps,
+    question3,
```

Comment:
The changes in this file were needed because: 

  * the `input-number` widget is deprecated 
  * the `numeric-input` widget does not have any validation it can do that does not require scoring data, so it can't be used for empty widget tests. 
  *